src/app/dspace/components/item/collection.component.ts

```diff
@@ -1,33 +1,28 @@
-import {Component, Input} from 'angular2/core';
+import {Component} from 'angular2/core';
+import {ROUTER_DIRECTIVES} from 'angular2/router';
 import {TranslatePipe} from "ng2-translate/ng2-translate";
 
-import {Collection} from "../../models/collection.model"
+import {Collection} from "../../models/collection.model";
 import {ViewElementComponent} from './view-element.component';
 
 /**
  * Component for the collections of the simple-item-view.
- * When you click on the collection name, it has to redirect to the right collection.
  */
 @Component({
     selector: 'item-collection',
-    inputs: ['itemData'],
-    directives: [ViewElementComponent],
+    inputs: ['itemParent'],
+    directives: [ROUTER_DIRECTIVES, ViewElementComponent],
     pipes: [TranslatePipe],
-    template:
-        `
-        <view-element [header]="componentTitle| translate">
-             <a [attr.href]="collectionURIPrefix+itemData.id">{{ itemData.name }}</a>
-        </view-element>
-        `
+    template: `
+                <view-element [header]="componentTitle| translate">
+                    <a *ngIf="itemParent && itemParent.component && itemParent.id" [routerLink]="[itemParent.component, {id: itemParent.id}]">{{ itemParent.name }}</a>
```

Comment:
This is a trivial example, as it is quite obvious what's going on. But in general, to improve readability and to keep as much logic as possible out of the templates it may be a good idea to put the contents of a complex (consisting of multiple parts) `*ngIf` in a method in the component class. e.g. `ngIf="hasValidParent()"`